MapboxMobileEvents/Reachability/MMEReachability.m

```diff
@@ -88,16 +83,13 @@ @implementation MMEReachability
 
 #pragma mark - Class Constructor Methods
 
-+(instancetype)reachabilityWithHostName:(NSString*)hostname
-{
++(instancetype)reachabilityWithHostName:(NSString*)hostname {
```

Comment:
The old brace style came from the iOS map SDK. Someday I want to rectify that in the map SDK too. (The macOS map SDK and shared “Darwin” code also inline braces as you’ve done here.)